packages/date-picker/src/_primitives/CalendarWrapper/CalendarWrapper.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { HTMLAttributes, useState } from "react"
+import { OverrideClassName } from "@kaizen/component-base"
+import classnames from "classnames"
+import { usePopper } from "react-popper"
+import { Calendar } from "../Calendar"
+import calendarWrapperStyles from "./CalendarWrapper.scss"
+
+interface CalendarWrapperProps
+  extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
+  children: React.ReactElement<typeof Calendar>
+  referenceElement: HTMLElement | null
+}
+
+export const CalendarWrapper: React.VFC<CalendarWrapperProps> = ({
```

Comment:
Not 100% sure if this is a good name. Was going to name it `FloatingCalendarWrapper` to give it more context, but it sounded a little weird too. Open to suggestions.

Reply:
Haha I think it's fair and still gives people a good idea of what it is - Should it be in a components folder under Calendar?